src/lint/validators.nim

```diff
@@ -75,9 +75,16 @@ proc isUrlLike(s: string): bool =
 const
   emptySetOfStrings = initHashSet[string](0)
 
+func isLowerKebab(s: string): bool =
+  ## Returns true if `s` is a lowercase and kebab-case string.
+  result = true
+  for c in s:
+    if c notin {'a'..'z', '0'..'9', '-'}:
```

Comment:
```suggestion
    if c in {'A'..'Z', '_', ' '}:
```
Do we want something like this instead? This allows special characters like `&`.

Reply:
No, the original code is what we want.